README.md

```diff
@@ -71,6 +71,7 @@ The core contributors so far are:
  * [Will Montgomery](https://github.com/csdigi)
  * [Fabian Siddiqi](https://github.com/FS89)
  * [Diana-Maria Costea](https://github.com/dianacostea)
+ * [Dimitar Pavlov](https://github.com/dimpavloff)
```

Comment:
There's a slightly more complete one on the website, don't mind Dimitar adding this for now as long as he doesn't mind if it goes away soon too :)